src/app/main/component/shared/components/events-list-item/events-list-item-modal/events-list-item-modal.component.ts

```diff
@@ -42,6 +43,10 @@ export class EventsListItemModalComponent implements OnInit, OnDestroy {
     this.bindLang(this.localStorageService.getCurrentLanguage());
   }
 
+  public starsHandler(index: number, value: number): void {
+    index > value ? (this.isEventRaited = true) : (this.isEventRaited = false);
```

Comment:
maybe write just ` this.isEventRaited = index > value;` ?

Reply:
Added changes according to comment